frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardPage/PublicOrEmbeddedDashboardPage.common.unit.spec.tsx

```diff
@@ -0,0 +1,137 @@
+import userEvent from "@testing-library/user-event";
+
+import { getIcon, screen, waitForLoaderToBeRemoved } from "__support__/ui";
+import { DASHBOARD_PDF_EXPORT_ROOT_ID } from "metabase/dashboard/constants";
+
+import { type SetupOpts, setup } from "./setup";
+
+const DASHBOARD_TITLE = '"My test dash"';
+
+const setupCommon = async (opts?: Partial<SetupOpts>) => {
+  return await setup({
+    ...opts,
+    dashboardTitle: DASHBOARD_TITLE,
+  });
+};
+
+describe("PublicOrEmbeddedDashboardPage", () => {
+  it("should display dashboard tabs", async () => {
+    await setupCommon({ numberOfTabs: 2 });
+
+    expect(screen.getByText("Tab 1")).toBeInTheDocument();
+    expect(screen.getByText("Tab 2")).toBeInTheDocument();
+  });
+
+  it("should display dashboard tabs if title is disabled (metabase#41195)", async () => {
+    await setupCommon({ hash: { titled: "false" }, numberOfTabs: 2 });
+
+    expect(screen.getByText("Tab 1")).toBeInTheDocument();
+    expect(screen.getByText("Tab 2")).toBeInTheDocument();
+  });
+
+  it("should display the header if title is enabled and there is only one tab", async () => {
```

Comment:
The test `should not display the header if title is disabled and there is only one tab (metabase#41393) and downloads are disabled` has been removed from both `*.common.*` and `*.enterprise.*` since `#downloads=false` only works when the token has `whitelabel` feature.